sdk/identity/identity-cache-persistence/test/internal/node/msalNodeTestSetup.ts

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Created a local copy of msalNodeTestSetup as I am not modernizing identity-cache-persistence at this time. I'd like to have tests running and green as a baseline so I would like to see #31660 resolved first